Teensy/src/gpio/gpio.cpp

```diff
@@ -50,7 +50,31 @@ static uint32_t gpio_getEngineRevs()
 
 static int32_t gpio_getDamperPosition()
 {
-    return 0;
+    // Defining pins //
+    #include <Arduino.h>
```

Comment:
Inclusions should always be at the top of the file. Also, you should avoid repeating inclusions. Even though duplicated inclusions should be protected by #ifdef statements, its a bad practice.